apple/testing/apple_test_rules.bzl

```diff
@@ -303,9 +308,18 @@ def _apple_ui_test_impl(ctx):
 
 
 apple_ui_test = rule(
-    _apple_ui_test_impl,
-    test=True,
+    implementation=_apple_ui_test_impl,
     attrs=_apple_ui_test_attributes(),
+    doc="""
+Rule to execute UI (XCUITest) tests for a generic Apple platform.
+
+Outputs:
+  test_bundle: The xctest bundle being tested. This is returned here as a
+      symlink to the bundle target in order to make it available for inspection
+      when executing `blaze build` on this target.
```

Comment:
s/blaze/bazel

Reply:
... 

it will be cleaned in a subsequent PR, sorry about that